subprojects/language-native/src/integTest/groovy/org/gradle/language/cpp/CppLibraryPublishingIntegrationTest.groovy

```diff
@@ -823,7 +823,7 @@ dependencies { implementation 'some.group:greeter:1.2' }
                 }
                 
                 components.withType(CppComponent) {
-                    targetMachines = [machines.host().x86(), machines.host().x86_64()]
+                    targetMachines = [machines.host().x86, machines.host().x86_64]
```

Comment:
⭕️ We should try to move away from those `host()` methods. I probably missed them last time. We can do that a separate PR.